CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ Familiarize yourself with how we do coding and documentation in the BigchainDB p
 
 ### Step 2 - Install some Dependencies
 
-Install MongoDB, Tendermint, and all of BigchainDB Server's dependencies. The [Quickstart page](https://docs.bigchaindb.com/projects/server/en/latest/quickstart.html) has some pointers. In fact, you could do everything in the Quickstart page short of installing BigchainDB with pip (since you will install from the source on GitHub), and you shouldn't run MongoDB or Tendermint yet.
+We recommend using `Docker` and `Docker Compose` based workflow for testing and development of bigchaindb. So the only dependencies you need to install are [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/) and [Docker Compose](https://docs.docker.com/compose/install/).
```

Comment:
The second sentence says "...to install are Docker and docker-compose and Docker Compose."

Reply:
+1, needs to be addressed.